Gruntfile.js

```diff
@@ -160,16 +160,16 @@
 				},
 				pushBump: {
 					command: [
-						"git add www/js/main.js config.xml package.json",
-						"git commit -m 'Base: Increment version number'",
+						"git add www/ config.xml package.json",
+						"git commit -m 'chore(vers): Increment version number'",
 						"git push"
 					].join( "&&" )
 				}
 			},
 
 			replace: {
 				about: {
-					src: [ "www/js/main.js" ],
+					src: [ "www/js/modules/ui-dom.js" ],
```

Comment:
This is a bit annoying. At some point we should change how the version number is injected into the app.

Reply:
Yes we could just import package json version maybe?